package.json

```diff
@@ -19,6 +19,9 @@
     "atom-linter": "^10.0.0",
     "atom-message-panel": "^1.2.4",
     "atom-package-deps": "^4.3.1",
+    "convert-source-map": "^1.5.0",
+    "glslify": "https://github.com/fand/glslify#sourcemaps",
```

Comment:
yea, this is just an experimental fork to get sourcemaps and disable minification.
I'm gonna replace it with glslify if it supports sourcemaps.